manifestgen.sh

```diff
@@ -42,7 +42,9 @@ CLI_SECTION=$(cat manifest.json | jq .cli)
 rm -f manifest.json
 
 SERVICES=(
+    "evmconnect"
     "ethconnect"
+    "evmconnect"
```

Comment:
One of these `evmconnect` entries should be removed

Reply:
will sort that now - might address the test coverage gap as I have to go round again